smart-contract-viem/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "type": "module",
+    "dependencies": {
+        "@noble/hashes": "^1.7.1",
+        "dotenv": "^16.4.1",
+        "rlp": "^3.0.0",
+        "viem": "2.7.3"
```

Comment:
prefer exact versions where possible. so remove the leading `^` from the version numbers specified here.